### DISCUSSION TITLE
What is the meaning of MERN?

### DISCUSSION THREAD

**Discussion by silvershiny** [Finalgoal231/support#10] [Q&A] [answered]
I am new to MERN stack.
Please explain about MERN is meaning.

**Comment by noahstoner** [comment, marked as answer]
Ok
You welcome.
MERN stands for MongoDB, Express, React, and Node.js, which are the four key technologies that make up the MERN stack. The MERN stack is a collection of open-source technologies used to build web and mobile applications.

**Comment by goldsunshines** [comment]
Yeah.
MERN - MongoDB, Express, React, Node.js
MEAN - MongoDB, Express, Angular, Node.js
MEVN - MongoDB, Express, Vue.js, Node.js

**Reply by MWaseemAkram07** [reply]
PERN - PostgreSQL, Express, React, Node.js

**Reply by goldsunshines** [reply]
That is something interesting.
Have you ever built the project like this?
Can you share the project source and live link here?

**Comment by MWaseemAkram07** [comment]
Yes, I have worked on a basic application using the PERN stack—a chat application built with PostgreSQL, Express, React, and Node.js. I used Prisma as the ORM and Tailwind CSS for styling. This project is more of an introductory setup rather than a full-fledged application, but it’s a great way to explore the core functionalities of the PERN stack and gain insights into how these technologies work together.Feel free to check out the repository here: GitHub Repository Link : "https://github.com/MWaseemAkram07/Pern-Stack-Chat-Application". I’d be happy to discuss any specifics or answer questions if you’re interested.

Yahoo Mail: Search, Organize, Conquer 
 
  On Wed, Oct 23, 2024 at 5:22 PM, Shiny ***@***.***> wrote:   


That is something interesting.
Have you ever built the project like this?
Can you share the project source and live link here?
